src/styles/blocks/card.scss

```
@@ -0,0 +1,106 @@
+.card {
+  width: $card-width;
+
+  border: $card-border-style;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  align-items: center;
+  background-color: $background-color-card;
+
+  &__img {
+    display: block;
+
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $card-main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    line-height: 14px;
+
+    font-size: 10px;
+    font-weight: 400;
+    color: #616070;
+
+    margin-bottom: 16px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+
+    &-text {
+      line-height: 14px;
+
+      font-size: 10px;
+      font-weight: 400;
+      color: $card-main-text-color;
+    }
```

if `-text` modifier use please `--text` 
Or use it as separate selector `&__rate-text `

---

When I tried to fix it i got this error:
C:\Progects\layout_catalog\src\index.html
 155:13  error  Tag doesn't have modifiable class  [card__rate--text]    (modifiable-class)
 159:13  error  Tag doesn't have modifiable class  [card__price--text]   (modifiable-class)
 161:13  error  Tag doesn't have modifiable class  [card__price--value]  (modifiable-class)
 but if I don`t tuch this, everything is good!
 Also I dded a transition to cards, now the hover is smooth

---

@DanilWeda probably means you should not nest `rate-text` within `rate` as it looks like an error
As long as `__rate` and `__rate-text` are two independent elements you should also have separate code sections for them

Something like this:
```suggestion
&__rate {
    display: flex;
    justify-content: space-between;

    margin-bottom: 24px;
}
&__rate-text {
      line-height: 14px;

      font-size: 10px;
      font-weight: 400;
      color: $card-main-text-color;
}
```

---

You have the same problem in other files